src/datreant/core/collections.py

```diff
@@ -439,6 +439,17 @@ def relpaths(self):
         return [member.relpath for member in self]
 
     @property
+    def parents(self):
```

Comment:
@kain88-de yes, but a ``Bundle`` by definition only works with ``Treant``s, since it gives access to things like ``Tags`` and ``Categories``, which only ``Treant``s can possess.